### PR TITLE
Refreshed Tumbleweed logo

### PR DESCRIPTION
**Pull request by lkocman**
* Details at gh#openSUSE/distribution-logos#10

ps. yes, the logo in the menu has undefined color :)

image
image

### CODE REVIEW

**Comment by lkocman** [comment]
@ddemaio I would not do that independently from wallpapers. All changes to branding wallpapers etc are currently in the same staging, so we'll enroll the change at once. It might make sense to introduce it all together with tumbleweed wallpaper set etc.

File: _includes/logos/tumbleweed.svg [hunk 1]
```diff
@@ -1 +1,86 @@
...
+     id="metadata230">
+    <rdf:RDF>
+      <cc:Work
+         rdf:about="">
+        <dc:format>image/svg+xml</dc:format>
+        <dc:type
+           rdf:resource="http://purl.org/dc/dcmitype/StillImage" />
+      </cc:Work>
+    </rdf:RDF>
+  </metadata>
```
Comment (hellcp) [inline _includes/logos/tumbleweed.svg:81]: Unnecessary
```suggestion
  >
```

File: _includes/logos/tumbleweed.svg [hunk 2]
```diff
@@ -1 +1,86 @@
...
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<svg
+   fill="currentColor"
+   version="1.1"
+   viewBox="0 0 256 256"
+   id="svg234"
+   sodipodi:docname="tumbleweed.svg"
+   inkscape:version="1.3.2 (091e20ef0f, 2023-11-25)"
+   xmlns:inkscape="http://www.inkscape.org/namespaces/inkscape"
+   xmlns:sodipodi="http://sodipodi.sourceforge.net/DTD/sodipodi-0.dtd"
```
Comment (hellcp) [inline _includes/logos/tumbleweed.svg:10]: Unnecessary
```suggestion
```

File: _includes/logos/tumbleweed.svg [hunk 3]
```diff
@@ -1 +1,86 @@
...
+     id="path1"
+     style="stroke-width:1.01652" />
```
Comment (hellcp) [inline _includes/logos/tumbleweed.svg:85]:
```suggestion
     stroke-width="1" />
```

File: assets/images/logos/tumbleweed.svg
Comment (hellcp) [inline assets/images/logos/tumbleweed.svg:1]: Can also be cleaned up

File: assets/images/logos/tumbleweed_header.svg
Comment (hellcp) [inline assets/images/logos/tumbleweed_header.svg:1]: Can also be cleaned up

File: assets/images/logos/tumbleweed.svg
Comment (lkocman) [reply assets/images/logos/tumbleweed.svg:1]: will do!

File: assets/images/logos/tumbleweed_header.svg
Comment (lkocman) [reply assets/images/logos/tumbleweed_header.svg:1]: will do!

**Pull request merged**